app/view_utils/path_helpers.rb

```diff
@@ -2,16 +2,17 @@ module PathHelpers
 
   module_function
 
-  def search_path(community, user, locale_param, opts = {})
+  def search_path(community_id:, user:, locale_param:, default_locale:, opts: {})
```

Comment:
This gets rid of the community model but user model is still passed as model. Could it be in the scope to change that too.